paddle/fluid/framework/paddle2cinn/cinn_cache_key.cc

```diff
@@ -39,13 +44,42 @@ CinnCacheKey::CinnCacheKey(const ir::Graph& graph,
   this->SetKey(graph, input_shapes, arch_str);
 }
 
+std::string CinnCacheKey::HashGraph(const ir::Graph& graph) {
```

Comment:
Hash usually return hash type. Can we call it `GraphHashString` or `GraphHashStr`?

Or you just return hash type at line 73 while let caller of this function make it to string.

Reply:
Done.Changed to return `size_t`.